packages/dashboard/src/karma/fixture.js

```diff
@@ -82,6 +82,7 @@ const defaultConfig = {
   cdnURL: 'https://cdn.example.com/',
   version: '1.0.0-alpha.9',
   archiveURL: 'https://example.com/',
+  archiveURLRaw: 'https://example.com/',
```

Comment:
Nit: Using different URLs here would make it easier to understand the difference

Reply:
Updated https://github.com/GoogleForCreators/web-stories-wp/pull/10414/commits/2636431014094256655ac3f9c46f1518176932b9